app/payment-request/payment-request-schema.js

```diff
@@ -0,0 +1,23 @@
+const Joi = require('joi')
+
+module.exports = Joi.object({
+  sourceSystem: Joi.string().required(),
```

Comment:
I'm starting to think we don't need this in the message as we can get the source from the service bus message metadata.  For now just change this to optional and we may remove it later.  ie `.allow('')`

Reply:
agree